src/errorHandler.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Alert, Clipboard } from 'react-native';
+import hathorLib from '@hathor/wallet-lib';
+import * as Sentry from '@sentry/react-native';
+import VersionNumber from 'react-native-version-number';
+
+/**
+ * Send error to Sentry
+ */
+export const sentryReportError = (error) => {
+  Sentry.init({
+    dsn: 'https://c1ebae9159f741e8937abdbfbeba8e8a@o239606.ingest.sentry.io/5304101',
+  });
+  Sentry.withScope((scope) => {
+    scope.setExtra('App version', JSON.stringify(VersionNumber));
+    Sentry.captureException(error);
+  });
+};
+
+/**
+ * Copy access data from storage to the clipboard.
+ * This could be useful to ask the user to copy it and check if
+ * all keys are correctly there.
+ * With this we prevent sending sensitive data to Sentry
+ */
+export const copyData = (error) => {
+  const accessData = JSON.stringify(hathorLib.wallet.getWalletAccessData());
+  Clipboard.setString(`${error.name} - ${error.message}\n${accessData}`);
+};
+
+/**
+ * Capture all JS exceptions that might happen and show an alert to the user
+ */
+export const errorHandler = (error, isFatal) => {
+  if (isFatal) {
+    Alert.alert(
+      'Unexpected error occurred',
+      '\nUnfortunately a fatal error happened and you need to restart your app.\n\nBefore doing that, we kindly ask you to report this error to the Hathor team clicking on the button below.',
+      [{
+        text: 'Report error',
+        onPress: () => {
+          sentryReportError(error);
+        }
+      },
+      {
+        text: 'Copy data',
```

Comment:
Maybe we should remove this button. Simply saying "Copy data" may mislead many users.

Reply:
As in the comment above, I agree we shouldn't allow users to copy sensitive data, so there is no reason to keep this button. Will remove it, okay?

Reply:
Should we replace it by a Cancel button? What's the user supposed to do if he doesn't want to send the info? Restart the app? Maybe a Close button then.

Reply:
For what I know Apple has never allowed apps to close themselves, that's why there are no react native commands to do it, only libraries (https://github.com/wumke/react-native-exit-app) that execute native commands. This Apple restriction was in the past, I don't know today. I could do some tests with this library but still no sure if it's worth it to risk having the app rejected by Apple.

Reply:
Got it, I agree to just remove the button then.

Reply:
Ok! Go ahead!

Reply:
Button deleted.